scripts/RMC/Adv. Ghost Comp/Load_data.py

```diff
@@ -0,0 +1,54 @@
+from dolphin import event, gui, utils
```

Comment:
The GUI module is not used in this file.
```suggestion
from dolphin import event, utils
```